lib/rules/at-rule-descriptor-no-unknown/README.md

```diff
@@ -0,0 +1,54 @@
+# at-rule-descriptor-no-unknown
+
+Disallow unknown descriptors for at-rules.
+
+<!-- prettier-ignore -->
+```css
+@counter-style example {
+  unknown-descriptor: value;
+/**                ↑
+ * Descriptors like this */
+}
+```
+
+This rule considers descriptors defined in the CSS Specifications, up to and including Editor's Drafts, to be known.
+
+The [`message` secondary option](../../../docs/user-guide/configure.md#message) can accept the arguments of this rule.
```

Comment:
```suggestion
The [`message` secondary option](../../../docs/user-guide/configure.md#message) can accept the arguments of this rule.

This rule checks descriptors within at-rules. To check properties, you can use the [property-no-unknown](../property-no-unknown/README.md) rule.
```

In line with https://github.com/stylelint/stylelint/issues/8150#issuecomment-2536062548